### DISCUSSION TITLE
Can the current Nerd Fonts replace Google's Material Design Icons font?

### DISCUSSION THREAD

**Discussion by Gitoffthelawn** [ryanoasis/nerd-fonts#1738] [Q&A] [answered]
For better or worse, many websites still use Google's Material Design Icons font.  These sites typically download that font to the user agent (e.g. local web browser) each time a user visits such a site.

To eliminate this extra internet traffic and lag, I often recommend installing the Material Design Icons font from one of the zip files [available here](https://github.com/google/material-design-icons/releases/tag/4.0.0), and having the user agent point to the locally installed font via a little CSS override.  (BTW, you can do this for all common Google fonts to really reduce traffic and speed things up.)

Can the current Nerd Fonts be used instead of Google's Material Design Icons font for this purpose?  Or will Nerd Fonts not work due to differing code points?

**Comment by Finii** [comment, marked as answer]
Nerd Fonts does not use Googles 'Material Icons' https://github.com/google/material-design-icons
Instead it uses/has always used the project Material Design https://github.com/Templarian/MaterialDesign
They claim _This repo also contains converted icons from Google's official icon set_, whatever that means.

![image](https://github.com/user-attachments/assets/d7bed537-7a83-420b-9912-a7448d17e159)

I never researched that / had a look at Google's project.

## Codepoints

We use the "upper" (5 digit) codepoints of the icons: `F0001` - `F1AF0`
The "lower" range has been dropped with `Nerd Fonts 3.0.0`:
https://github.com/ryanoasis/nerd-fonts/blob/master/changelog.md#breaking-2-material-design-icons-codepoints

I looked into the font files of Google's project, and they seem to have clung to the lower 4 digit codes, which is a complete mess (sorry, imho only of course), because there are so many icons and they are spread out which is a nightmare if you want to combine this with other icon projects like we do.

So in conclusion, the answer is most likely: No. Sorry.

**Reply by Gitoffthelawn** [reply]
Thanks.  That's exactly what I wanted to know.
